packages/eslint-plugin/src/rules/index.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable eslint-comments/no-use */
+/* eslint sort-keys: error */
```

Comment:
I sorted the rules alphabetically like always and also added this eslint rule just for this file so it's enforced. The imports still can be out of order. There's an `eslint-plugin-import` rule which could enforce that.

Tell me what you think.